.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode/
```

Comment:
There will not be `node_modules` because `node` and `npm` is not involved: instead of install dependencies, just `deno test` or `deno bundle`.

Please let me know if this is acceptable.